hash/msx2_flop.xml

```diff
@@ -10240,12 +12412,16 @@ The following floppies came with the machines.
 		<description>Schwarzschild II (Japan)</description>
 		<year>1990</year>
 		<publisher>Kogado</publisher>
+		<info name="alt_title" value="シュヴァルツシルト II 帝國ノ背信"/>
```

Comment:
The subtitle is in the `alt_title` but not the description for this game now.

Reply:
Also added the subtitle to the description now.